<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Library DBMS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>600</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1000</width>
      <height>600</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Search</string>
     </attribute>
     <widget class="QPushButton" name="searchbks">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Search for books</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchmbr">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Search for member</string>
      </property>
     </widget>
     <widget class="QLabel" name="sbks">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter book id or book name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sbksinput">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sbkssubmit">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>80</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLabel" name="smbr">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>431</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter Member id, email id or phone no.</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tblbooks">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>941</width>
        <height>261</height>
       </rect>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <column>
       <property name="text">
        <string>Book id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Book Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>No. of copies in total</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>No. of copies issued</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="clrtable">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>400</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear Table</string>
      </property>
     </widget>
     <widget class="QPushButton" name="showbks">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>20</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Show all books</string>
      </property>
     </widget>
     <widget class="QPushButton" name="showmbr">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>20</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Show all members</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tblissue">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>941</width>
        <height>261</height>
       </rect>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <column>
       <property name="text">
        <string>Issuer's Person id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name of the Person</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Book id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Book Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Copy no. of the book</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Issue Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Due Date</string>
       </property>
      </column>
     </widget>
     <widget class="QTableWidget" name="tblmbr">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>941</width>
        <height>261</height>
       </rect>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <column>
       <property name="text">
        <string>Person id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Email id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Contact No.</string>
       </property>
      </column>
     </widget>
     <widget class="QLineEdit" name="smbrinput">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="smbrsubmit">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>80</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tblmbrinfo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>941</width>
        <height>261</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Person id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Email id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Contact no.</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name of book issued</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Book id</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="Issuetab">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <attribute name="title">
      <string>Issue Book</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1000</width>
        <height>1000</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="Enteremail">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>361</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Enter your id no., email id or phone no.</string>
       </property>
      </widget>
      <widget class="QPushButton" name="emailsubmit">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>20</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
      <widget class="QLabel" name="enterbookid">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Enter Book Id</string>
       </property>
      </widget>
      <widget class="QPushButton" name="bkidsubmit">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>100</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
      <widget class="QLabel" name="bookerror">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>291</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>No such book found in database.</string>
       </property>
      </widget>
      <widget class="QLabel" name="enterbname">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Enter Book Name</string>
       </property>
      </widget>
      <widget class="QPushButton" name="bnamesubmit">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>210</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="emailinput">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>20</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="bkidinput">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="bname">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>210</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="copy">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>290</y>
         <width>561</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>How many copies of this book are there in total?</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="copyinput">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>290</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="copysubmit">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>290</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Returntab">
     <attribute name="title">
      <string>Return Book</string>
     </attribute>
     <widget class="QLabel" name="enterbookid2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter Book Id</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="bkidinput2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="bkidsubmit2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>100</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLabel" name="enterid">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>351</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter your id no., email id or phone no.</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idinput">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="idsubmit">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>20</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="newmem">
     <attribute name="title">
      <string>Register New Member</string>
     </attribute>
     <widget class="QLabel" name="entername">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter your Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameinput">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="namesubmit">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLabel" name="enter_email">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter your email id</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="emailinput2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="emailsubmit2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>90</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLabel" name="enterphone">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter your phone no.</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="phoneinput">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>170</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="phonesubmit">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>170</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="newbook">
     <attribute name="title">
      <string>Register New Book</string>
     </attribute>
     <widget class="QLabel" name="enterbookid3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter Book Id</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="bkidinput3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="bkidsubmit3">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLabel" name="enterbname2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter Book Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="bnameinput">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="bnamesubmit2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>100</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLabel" name="copy2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>561</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>How many copies of this book are there in total?</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="copyinput2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>180</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="copysubmit2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>180</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1011</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>emailinput</tabstop>
  <tabstop>emailsubmit</tabstop>
  <tabstop>bkidinput</tabstop>
  <tabstop>bkidsubmit</tabstop>
  <tabstop>bname</tabstop>
  <tabstop>bnamesubmit</tabstop>
  <tabstop>bkidinput2</tabstop>
  <tabstop>bkidsubmit2</tabstop>
  <tabstop>nameinput</tabstop>
  <tabstop>namesubmit</tabstop>
  <tabstop>emailinput2</tabstop>
  <tabstop>emailsubmit2</tabstop>
  <tabstop>phoneinput</tabstop>
  <tabstop>phonesubmit</tabstop>
  <tabstop>bkidinput3</tabstop>
  <tabstop>bkidsubmit3</tabstop>
  <tabstop>bnameinput</tabstop>
  <tabstop>bnamesubmit2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
